### DISCUSSION TITLE
post/model-serving-framework/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#32] [Announcements]
# post/model-serving-framework/

카카오페이에서 효율적인 모델 서빙을 위해 프레임워크를 도입한 과정과 프레임워크의 성능 최적화를 위해 테스트한 결과를 공유합니다.

https://tech.kakaopay.com/post/model-serving-framework/

**Comment by kyeongchan92** [comment]
잘 읽었습니다. 유용한 정보 감사합니다!

**Comment by myrhymetree** [comment]
잘 모르던 주제였는데 흥미롭게 읽었습니다.

**Comment by Jungjihyuk** [comment]
안녕하세요~ 글이 너무 읽기 편하고 정리가 잘 되어 있네요! 
좋은글 감사합니다 ㅎㅎ 
궁금한점이 한 가지 있는데요! 
Triton이 RPS가 다른 것에 비해 3배 이상 높은 반면에 latency는 다른 것에 비해 높은 편이네요!? 
Triton에서 처리한 결과값을 클라이언트에 전달할때 추가적인 변환과정이 있어서 latency가 높은건가요?? 
무엇때문에 latency가 높아지는지 궁금합니다~!
